src/components/RiskComponentDropdown.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <v-expansion-panel class="riskComponentDropdown">
+    <v-expansion-panel-header class="d-flex flex-row">
+      <div :class="riskClass" class="componentIconContainer" flex-shrink-1>
+        <CrowdingIcon v-if="isCrowding" class="componentIcon" />
+        <DropletsIcon v-if="isDroplets" class="componentIcon" />
+        <TimeIcon v-if="isTime" class="componentIcon" />
+        <VentIcon v-if="isVent" class="componentIcon" />
+      </div>
+      <div class="componentHeaderText" flex-grow-1>
+        <div class="componentTitle">{{ title }}</div>
+        <div class="componentRiskLabel">{{ risk }}</div>
+      </div>
+    </v-expansion-panel-header>
+    <v-expansion-panel-content>
+      <Markdown :source="notes" />
+    </v-expansion-panel-content>
+  </v-expansion-panel>
+</template>
+
+<script>
+import { mapGetters } from "vuex";
+import CrowdingIcon from "@/assets/Risk_Crowding-filled.svg";
+import DropletsIcon from "@/assets/Risk_Droplets-filled.svg";
+import TimeIcon from "@/assets/Risk_Time-filled.svg";
+import VentIcon from "@/assets/Risk_Ventilation-filled.svg";
+import Markdown from "vue-markdown";
+
+export default {
+  components: {
+    CrowdingIcon,
+    DropletsIcon,
+    TimeIcon,
+    VentIcon,
```

Comment:
could these be moved up to RiskComponents and then the correct icon could be passed down as a prop?

Reply:
Any idea how I could do that?

Reply:
I tried to put the icons in RiskComponents and pass them individually, but it's causing all sorts of errors. 
```
vue.runtime.esm.js?2b0e:619 [Vue warn]: Property or method "DropletsIcon" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. See: https://vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.

found in

---> <RiskComponents> at src/components/RiskComponents.vue
       <RiskDescription> at src/components/RiskDescription.vue
         <SearchResults> at src/components/SearchResults.vue
           <SearchResults> at src/views/SearchResults.vue
             <AppBody> at src/components/AppBody.vue
               <VMain>
                 <VApp>
                   <App> at src/App.vue
                     <Root>
```